apps/ai-image-generator/vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+
+export default defineConfig(() => ({
+  base: '', // relative paths
+  server: {
+    port: 3000,
+  },
```

Comment:
Here is the defineConfig  we are using in AICG that works with prod (assuming you go with using @emotion/react):

```
import react from '@vitejs/plugin-react';
import path from 'path';
import { defineConfig, splitVendorChunkPlugin } from 'vite';
import { viteCommonjs } from '@originjs/vite-plugin-commonjs';

export default defineConfig(() => ({
  base: '', // relative paths
  server: {
    port: 3000,
  },
  preview: {
    port: 3000,
  },
  plugins: [react({ jsxImportSource: '@emotion/react' }), viteCommonjs(), splitVendorChunkPlugin()],
  resolve: {
    alias: {
      '@components': path.resolve(__dirname, './src/components'),
      '@configs': path.resolve(__dirname, './src/configs'),
      '@hooks': path.resolve(__dirname, './src/hooks'),
      '@locations': path.resolve(__dirname, './src/locations'),
      '@providers': path.resolve(__dirname, './src/providers'),
      '@utils': path.resolve(__dirname, './src/utils'),
    },
  },
  test: {
    environment: 'happy-dom',
  },
}));
```

This also makes the preview command's port set to the same as server, which will allow you to easily test it in the dev environment (this defaults to a different port)